### ISSUE TITLE
building from scratch errors

### ISSUE THREAD

**Issue by abhiboost** [snowballstem/pystemmer#20]
I grabbed the latest snowball module and built it from scratch. 
After that i copied the whole snowball folder to pystemmer. The installation succeeds this way and i have the new stemmers,. however, the import gives the following error:

```
removing 'PyStemmer-1.3.0' (and everything under it)
**********************************************************************
File "docs/quickstart.txt", line 8, in quickstart.txt
Failed example:
    import Stemmer
Exception raised:
    Traceback (most recent call last):
      File "/usr/lib/python2.7/doctest.py", line 1315, in __run
        compileflags, 1) in test.globs
      File "<doctest quickstart.txt[0]>", line 1, in <module>
        import Stemmer
    ImportError: /home/ubuntu/boostai/pystemmer/dist/PyStemmer-1.3.0/Stemmer.so: undefined symbol: arabic_UTF_8_create_env

```

any ideas how to fix it? im basically trying to use the unreleased stemmers and they compile okay in the snowball module.

**Comment by ojwb** [comment]
Sounds like you aren't linking in the arabic stemmer.

**Comment by ojwb** [comment]
I've just updated the repo to wrap the recently released Snowball 2.0.0 so closing this now.

**Comment by abhiboost** [comment]
Thank you.
